src/pages/kandidat/Kandidatdetaljer.tsx

```diff
@@ -49,22 +51,21 @@ const Kandidatdetaljer: FunctionComponent<Props> = ({ match, history, iEndremodu
     };
 
     const sjekkSkrivetilgang = async () => {
-        const harTilgang = await hentSkrivetilgang(fnr);
+        const harTilgang = await hentSkrivetilgang(aktorId);
         settSkrivetilgang(harTilgang);
     };
 
     useEffect(() => {
-        redirectVedUgyldigFnr();
         hentKandidatEllerVisFeilmelding();
         sjekkSkrivetilgang();
         // TODO: Fiks det underliggende problemet i stedet for å disable linting
-    }, [fnr]); // eslint-disable-line
+    }); // eslint-disable-line
```

Comment:
Mangler en tom array her for å hindre at useEffecten kjøres i evig loop

Reply:
Ser at vi hadde fnr som dependency i denne useEffecten før. Kanskje vi må ha aktørId som dependency her nå da?